Gemfile

```diff
@@ -9,59 +9,61 @@ unless dependencies.detect { |d| d.name == "manageiq-providers-amazon" }
+gem "aws-sdk",                        "~>2.2.19",      :require => false
 gem "color",                          "~>1.8"
-gem "dalli",                          "~>2.7.4",   :require => false
+gem "dalli",                          "~>2.7.4",       :require => false
 gem "default_value_for",              "~>3.0.2"
-gem "elif",                           "=0.1.0",    :require => false
+gem "elif",                           "=0.1.0",        :require => false
 gem "fast_gettext",                   "~>1.1.0"
-gem "fog-google",                     "~>0.3.0",   :require => false
-gem "fog-vcloud-director",            "~>0.1.1",   :require => false
+gem "fog-google",                     "~>0.3.0",       :require => false
+gem "fog-vcloud-director",            "~>0.1.1",       :require => false
 gem "gettext_i18n_rails",             "~>1.7.2"
 gem "gettext_i18n_rails_js",          "~>1.1.0"
-gem "google-api-client",              "~>0.8.6",   :require => false
-gem "hamlit",                         "~>2.0.0",   :require => false
+gem "google-api-client",              "~>0.8.6",       :require => false
+gem "hamlit",                         "~>2.0.0",       :require => false
 gem "hamlit-rails",                   "~>0.1.0"
 gem "high_voltage",                   "~>2.4.0"
-gem "inifile",                        "~>3.0",     :require => false
+gem "inifile",                        "~>3.0",         :require => false
 gem "jbuilder",                       "~>2.5.0" # For the REST API
-gem "mime-types",                     "~>2.6.1",   :require => "mime/types/columnar"
+gem "mime-types",                     "~>2.6.1",       :require => "mime/types/columnar"
 gem "nakayoshi_fork",                 "~>0.0.3"  # provides a more CoW friendly fork (GC a few times before fork)
-gem "net-ldap",                       "~>0.7.0",   :require => false
-gem "net-ping",                       "~>1.7.4",   :require => false
-gem "net-ssh",                        "=3.2.0",    :require => false
-gem "net_app_manageability",          ">=0.1.0",   :require => false
+gem "net-ldap",                       "~>0.7.0",       :require => false
+gem "net-ping",                       "~>1.7.4",       :require => false
+gem "net-ssh",                        "=3.2.0",        :require => false
+gem "net_app_manageability",          ">=0.1.0",       :require => false
 gem "novnc-rails",                    "~>0.2"
-gem "omniauth",                       "~>1.3.1",   :require => false
+gem "omniauth",                       "~>1.3.1",       :require => false
 gem "omniauth-google-oauth2",         "~>0.2.6"
-gem "open4",                          "~>1.3.0",   :require => false
+gem "open4",                          "~>1.3.0",       :require => false
 gem "outfielding-jqplot-rails",       "= 1.0.8"
-gem "ovirt_metrics",                  "~>1.2.0",   :require => false
+gem "ovirt-engine-sdk",               "~>4.0.0.alpha", :require => false # Required by the oVirt provider
```

Comment:
This is the only new line, the others are modified to keep the column alignment.